invenio_rdm_records/resources/config.py

```diff
@@ -421,6 +438,37 @@ class RDMParentGrantsResourceConfig(RecordResourceConfig, ConfiguratorMixin):
     error_handlers = grants_error_handlers
 
 
+class RDMGrantUserAccessResourceConfig(RecordResourceConfig, ConfiguratorMixin):
+    """Record grants user access resource configuration."""
+
+    blueprint_name = "record_user_access"
+
+    url_prefix = "/records/<pid_value>/access"
+
+    routes = {
+        "item": "/users/<subject_id>",
+        "list": "/users",
+    }
+
+    links_config = {}
+
+    request_view_args = {
+        "pid_value": ma.fields.Str(),
+        "subject_id": ma.fields.Str(),  # user id
+    }
+
+    grant_subject_type = "user"
```

Comment:
For the future `access/groups` endpoint - planning to reuse `RDMGrantsAccessResource` resource, but register it with 2 different recourse configs - 1 for `grant_subject_type = "user"` another for `grant_subject_type = "role"`

Reply:
Here it is: https://github.com/inveniosoftware/invenio-rdm-records/issues/1672